src/Symfony/Component/HttpKernel/ControllerMetadata/ControllerMetadata.php

```diff
@@ -0,0 +1,112 @@
+<?php
+namespace Symfony\Component\HttpKernel\ControllerMetadata;
+
+use Symfony\Component\HttpKernel\ControllerMetadata\Configuration\ConfigurationInterface;
+
+/**
+ * Responsible for storing metadata of a controller action.
+ *
+ * @author Iltar van der Berg <kjarli@gmail.com>
+ */
+class ControllerMetadata implements \Serializable
+{
+    private $className;
+    private $method;
+    private $arguments;
+    private $configurations;
+
+    /**
+     * @param string                   $className
+     * @param string                   $method
+     * @param ArgumentMetadata[]       $arguments
+     * @param ConfigurationInterface[] $configurations
+     */
+    public function __construct($className, $method, array $arguments, array $configurations)
+    {
+        $this->className = $className;
+        $this->method = $method;
+        $this->arguments = $arguments;
```

Comment:
@bghazy we got it... PHP-CS-Fixer is a nice tool.

But please, don't affirm your opinion as the only possible reality, this one can be wrong. 
Especially In this case, you are blaming a code part that is totally valid. Equals should not be aligned, neither in the Symfony core nor in a Symfony application, even via php-cs-fixer (the `align_equals` fixer is not part of the symfony level, but `unalign_equals` is).